ponyo/utils.py

```diff
@@ -11,3 +16,154 @@ def read_config(filename):
         items = lines.split("\t", 1)
         config_dict[items[0]] = eval(items[1])
     return config_dict
+
+
+def setup_dir(config_file):
+    """
+    Create directories to store files created by VAE training and
+    simulation analysis
+
+    Arguments
+    ----------
+    config_file: str
+        File containing user defined parameters
+    """
+
+    base_dir = os.path.abspath(os.path.join(os.getcwd(), "../"))
+
+    # Read in config variables
+    params = read_config(config_file)
+
+    # Load parameters
+    local_dir = params["local_dir"]
+    dataset_name = params["dataset_name"]
+    train_architecture = params["NN_architecture"]
+
+    # Create VAE directories
+    output_dirs = [
+        os.path.join(base_dir, dataset_name, "models"),
+        os.path.join(base_dir, dataset_name, "logs"),
+    ]
+
+    # Check if analysis output directory exist otherwise create
+    for each_dir in output_dirs:
+        if not os.path.exists(each_dir):
+            print("creating new directory: {}".format(each_dir))
+            os.makedirs(each_dir, exist_ok=True)
+
+        # Check if NN architecture directory exist otherwise create
+    for each_dir in output_dirs:
+        new_dir = os.path.join(each_dir, train_architecture)
+        if not os.path.exists(new_dir):
+            print("creating new directory: {}".format(new_dir))
+            os.makedirs(new_dir, exist_ok=True)
+
+    # Create results directories
+    output_dirs = [os.path.join(base_dir, dataset_name, "results")]
```

Comment:
This list includes only one element. The `for` loops on lines 65 and 71 seem redundant. If you do want to use `for` loop, these two loops can be merged into one.